pkg/controller/virtualmachinevolumeexport/local_pod.go

```diff
@@ -0,0 +1,104 @@
+package virtualmachinevolumeexport
+
+import (
+	"context"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+	hc "kubevirt-image-service/pkg/apis/hypercloud/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func (r *ReconcileVirtualMachineVolumeExport) syncLocalPod() error {
+	completed, found, err := r.isPvcCompleted()
```

Comment:
Do we need a local Pod? or just not delete the exporter pod after complete. Which one looks better?

Reply:
When the pod is complete, we will not be able to access it.